### ISSUE TITLE
Add a new CNAME for this project

### ISSUE THREAD

**Issue by MadhavBahlMD** [CodeToExpress/dailycodebase#353]
Will add a new CNAME for this project

**Comment by aaditkamat** [comment]
I thought that this would be included under a new domain name for the organization CodeToExpress. How about `codetoexpress.tech/dailycodes`?

**Comment by MadhavBahlMD** [comment]
@aaditkamat , yes we can use it as a sub directory or a sub domain for time being,

But I really want this project to have it's own domain, so if you know any free domain providers do mention 😁

Something like dailycodes.tech would look cool, but .tech is paid
